examples/experimental/dagster-airlift/dagster_airlift/core/sensor/builder.py

```diff
@@ -132,13 +132,18 @@ def airflow_dag_sensor(context: SensorEvaluationContext) -> SensorResult:
                 end_date_lte=None,
                 dag_query_offset=0,
             )
+        updated_asset_events = (
+            list(event_transformer_fn(context, airflow_data, all_asset_events))
+            if event_transformer_fn
+            else all_asset_events
+        )
```

Comment:
as we are calling into user code we should be far more paranoid with the returned values use a `user_code_error_boundary`

Reply:
One could argue that the `user_code_error_boundary` might be overkill, as the sensor is already in "user space". But at a minimum introspect on return values out of `event_transformer_fn`